translations/LICENSE_fr-FR

```diff
@@ -1,11 +1,11 @@
 License publique GLWT (Bonne Chance Avec Ça)
 (c) Tout droits réservés au monde entier, à l'exception du ou des auteurs.
 
-Chacun est autorisé à copier, distribuer, modifier, fusionner, vendre, publier, sous-licencier 
-ou faire tout ce qu'il désire avec ce logiciel, mais à ses PROPRES RISQUES ET PERILS.
+Chacun est autorisé à copier, distribuer, modifier, fusionner, vendre, publier, 
+sous-licencer ou ce qu'il désire avec ce logiciel, mais à ses PROPRES RISQUES ET PERILS.
```

Comment:
Something isn't right with this sentence, we should keep the older one.

Reply:
Hum yes you're right... Sorry